### PR TITLE
US137872 - grey colour updates

### PR DESCRIPTION
**Pull request by annabelsegalld2l**
This PR makes the following colour changes:

- Corundum - #b5bdc2, rgba(181, 189, 194, 0.2) -> #B1B9BE, rgba(177, 185, 190, 0.2). I used a hex-> rgba calculator for this one. 
- Chromite- #9ea5a9 -> #90989d
- Galena - #868c8f -> #6e7477
- Tungsten - #6e7376 -> #494C4E
- Ferrite - #494c4e -> #202122

Note that icons do not need to be changed, since they are now using Tungsten, which is the old Ferrite.

### CODE REVIEW

**Comment by dlockhart** [comment]
You'll need to update this line so that icons don't shift their color:
https://github.com/BrightspaceUI/core/blob/main/components/icons/icon-styles.js#L8

That should remove some of the visual-diffs with icons.

File: components/dropdown/dropdown-content-styles.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const dropdownContentStyles = css`
 	:host([theme="dark"]) {
 		--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation-dark;
 		--d2l-dropdown-animation-name: d2l-dropdown-animation-dark;
-		--d2l-dropdown-background-color: #333536; /* ferrite @ 70% */
+		--d2l-dropdown-background-color: #202122b3; /* ferrite @ 70% */
```
Comment (dlockhart) [inline components/dropdown/dropdown-content-styles.js:27]: I think there's a mistake in this HEX code -- should only have 6 characters.

File: components/menu/menu.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ class Menu extends ThemeMixin(HierarchicalViewMixin(LitElement)) {
 			}
 
 			:host([theme="dark"]) {
-				--d2l-menu-background-color: #333536; /* ferrite @ 70% */
+				--d2l-menu-background-color: #202122b3; /* ferrite @ 70% */
```
Comment (dlockhart) [inline components/menu/menu.js:68]: Ditto here.

File: components/button/button-icon.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 					--d2l-button-icon-min-height: calc(2rem + 2px);
 					--d2l-button-icon-min-width: calc(2rem + 2px);
 					--d2l-button-icon-h-align: calc(((2rem + 2px - 0.9rem) / 2) * -1);
-					--d2l-icon-fill-color: var(--d2l-button-icon-fill-color, var(--d2l-color-ferrite));
+					--d2l-icon-fill-color: var(--d2l-button-icon-fill-color, var(--d2l-color-tungsten));
```
Comment (annabelsegalld2l) [inline components/button/button-icon.js:63]: Icon fill will stay the same, so there should be no icon visual-diff changes.

File: components/button/button-icon.js [hunk 4]
```diff
@@ -75,7 +75,7 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 				}
 				:host([theme="dark"]) {
 					--d2l-button-icon-background-color: transparent;
-					--d2l-button-icon-background-color-hover: rgba(51, 53, 54, 0.9); /* ferrite @70% @90% */
+					--d2l-button-icon-background-color-hover: rgba(51, 53, 54, 0.9); /* tungsten @70% @90% */
```
Comment (annabelsegalld2l) [inline components/button/button-icon.js:78]: No dark mode changes (new Ferrite is too dark, not contrasted with background)

File: components/menu/menu.js [hunk 5]
```diff
@@ -65,12 +65,12 @@ class Menu extends ThemeMixin(HierarchicalViewMixin(LitElement)) {
...
 			:host([theme="dark"]) {
-				--d2l-menu-background-color: #333536; /* ferrite @ 70% */
+				--d2l-menu-background-color: #333536; /* tungsten @ 70% */
 				--d2l-menu-background-color-hover: #123559; /* celestine-1 @ 50% */
-				--d2l-menu-border-color: var(--d2l-color-ferrite);
+				--d2l-menu-border-color: var(--d2l-color-tungsten);
 				--d2l-menu-foreground-color: var(--d2l-color-sylvite);
 				--d2l-menu-foreground-color-hover: #ffffff;
-				--d2l-menu-separator-color: var(--d2l-color-tungsten);
+				--d2l-menu-separator-color: var(--d2l-color-galena);
```
Comment (annabelsegalld2l) [inline components/menu/menu.js:73]: Changed this one lighter to Galena so we get a different colour for the separator.

**Review by annabelsegalld2l — COMMENTED** [review]
I think this is ready for review. Will need a comb-through of the visual diffs to ensure everything looks as expected.

**Review by dlockhart — APPROVED** [review]
LGTM!

**Pull request merged**